quiz-app/README.md

```diff
@@ -1,35 +1,118 @@
 # Quiz app
 
-This quiz app helps you display questions and answers in a quiz format. 
+These quizzes are the pre- and post-lecture quizzes for the data science curriculum at https://aka.ms/webdev-beginners
 
-## Install the app
+## Adding a translated quiz set
 
-Run `npm install` to install the dependencies.
+Add a quiz translation by creating matching quiz structures in the `assets/translations` folders. The canonical quizzes are in `assets/translations/en`. The quizzes are broken into several groupings. Make sure to align the numbering with the proper quiz section. There are 40 quizzes total in this curriculum, with the count starting at 0.
```

Comment:
```suggestion
Add a quiz translation by creating matching quiz structures in the `assets/translations` folders. The canonical quizzes are in `assets/translations/en`. The quizzes are broken into several groupings. Make sure to align the numbering with the proper quiz section. There are 40 quizzes total in this curriculum, with the count starting at 0.

Here's the shape of a translation file:

```json
[
	{
		"title": "A title",
		"complete": "A complete button title",
		"error": "An error message upon selecting the wrong answer",
		"quizzes": [
			{
				"id": 1,
				"title": "Title",
				"quiz": [
					{
						"questionText": "The question asked",
						"answerOptions": [
							{
								"answerText": "Option 1 title",
								"isCorrect": "true"
							},
							{
								"answerText": "Option 2 title",
								"isCorrect": "false"
							}
						]
					},
					}
				]
			}
		]
```

```